docs/Operators.md

```diff
@@ -3520,40 +3520,43 @@ for from_type, to_type in test_cases:
                 "INF",
                 "+INF",
                 "-INF",
+                "-0.0000001",
```

Comment:
Just clarifying - is it just `-0.0000001` or do we do the same for `-0.1`, `-0.4` etc?

Reply:
-0.0000001 is rounded to 0 but before this fix, it was -0->nan and not +0 which was wrong

Reply:
should any other negative values be rounded to +0 too?

Reply:
Not `-0.1`, but some other values, yes ... that's my understanding. The [documentation](https://onnx.ai/onnx/technical/float8.html#e4m3fnuz-and-e5m2fnuz) mentions the `Min` value representable in the format ... eg., 2^-17 for E5M2 variant. I guess values in the range [-Min, 0] will be rounded to 0? Xavier can clarify.